Add Deploy, Deploy Contrib and Forms Deploy 15.0.0-rc1 release notes (#6561)
umbraco/UmbracoDocs: 15/umbraco-deploy/upgrades/version-specific.md

---

# Version Specific Upgrade Details
This article provides specific upgrade documentation for migrating to Umbraco Deploy version 15.

{% hint style="info" %}
If you are upgrading to a minor or patch version, you can find the details about the changes in the [Release Notes](../release-notes.md) article.
{% endhint %}

## Version Specific Upgrade Notes History
Version 15 of Umbraco Deploy has a minimum dependency on Umbraco CMS core of `15.0.0`. It runs on .NET 9.

### **Breaking changes**

> Version 15 contains some breaking changes. Not many projects are expected to be affected by them, as they are in relevant areas when extending Deploy to support additional entities and/or property editors. For reference though, the full details are listed here:

Reviewer: Suggestion to shorten a sentence, and avoid mentioning anything about how many will be affected - I don't think this is relevant 🙌 

```suggestion
Version 15 contains some breaking changes. The breaking changes appear in areas related to extending Deploy to support additional entities and property editors. For reference, the full details are listed here:
```

---

# Version Specific Upgrade Details

This article provides specific upgrade documentation for migrating to Umbraco Deploy version 15.

{% hint style="info" %}
If you are upgrading to a minor or patch version, you can find the details about the changes in the [Release Notes](../release-notes.md) article.
{% endhint %}

## Version Specific Upgrade Notes History

Version 15 of Umbraco Deploy has a minimum dependency on Umbraco CMS core of `15.0.0`. It runs on .NET 9.

### Breaking changes

> Version 15 contains breaking changes. Not many projects are expected to be affected by them, as they are in relevant areas when extending Deploy to support additional entities and/or property editors. For reference though, the full details are listed here:

Reviewer: Suggestion to shorten a sentence, and avoid mentioning anything about how many will be affected - I don't think this is relevant 🙌

```suggestion
Version 15 contains breaking changes. The breaking changes appear in areas related to extending Deploy to support additional entities and property editors. For reference though, the full details are listed here:
```
